### ISSUE TITLE
[model_to_estimator] summary in keras loss/metrics function get duplicate scalars with same value in TensorBoard

### ISSUE THREAD

**Issue by Jasonnor** [tensorflow/tensorflow#32603]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  Linux Ubuntu 16.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.13.1
- Python version: 3.5.2
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: CUDA 10.0
- GPU model and memory: 

**Describe the current behavior**
Using tf.summary.scalar in keras loss/metrics function, then use model_to_estimator to convert keras model to estimator, tensorboard get duplicate scalars with totally same value. (e.g., `metrics/accuracy/train` and `metrics/accuracy/train_1`)

**Describe the expected behavior**
It should only summary one scalar.

**Code to reproduce the issue**
A example here:
```python
    ...

    model = tf.keras.Model(inputs=input_layer, outputs=[output_layer1, output_layer2])

    def accuracy(y_true, y_pred):
        y_pred, y_true = reshape_keras_tensor(y_pred, y_true)
        acc = tf.reduce_mean(
            tf.cast(tf.equal(tf.cast(y_true, tf.int64), tf.argmax(input=y_pred, axis=1)), tf.float32))
        # Where I summary, without this line, it will not summary anything during training
        tf.summary.scalar('train', acc)
        return acc

    def total_loss(y_true, y_pred):
        y_pred, y_true = reshape_keras_tensor(y_pred, y_true)
        loss =  tf.nn.sparse_softmax_cross_entropy_with_logits(
            labels=y_true,
            logits=y_pred)
        loss = tf.reduce_mean(cross_entropy)
        # Where I summary, without this line, it will not summary anything during training
        tf.summary.scalar('train', loss)
        return loss

    ...

    model.compile(optimizer=optimizer_builder.build(config.optimizer),
                  loss={output_layer1: total_loss},
                  metrics={output_layer1: accuracy})

    estimator = tf.keras.estimator.model_to_estimator(
        keras_model=model,
        config=run_config,
    )
    train_spec = tf.estimator.TrainSpec(
        input_fn=lambda: train_input_fn(config))
    eval_spec = tf.estimator.EvalSpec(
        input_fn=lambda: eval_input_fn(config))
    tf.estimator.train_and_evaluate(estimator, train_spec, eval_spec)
```

**Other info / logs**
TensorBoard with duplicate scalars:
![2019-09-18 10 28 17](https://user-images.githubusercontent.com/7609173/65102919-27da0c00-d9ff-11e9-8a46-6d00ea39a4eb.png)
![2019-09-18 10 27 56](https://user-images.githubusercontent.com/7609173/65102920-2872a280-d9ff-11e9-9d85-b626593688a7.png)

**Comment by nfelt** [comment]
You can just edit the `tf.summary.scalar('train', ...)` calls to pass in different string tags - right now they're both using `train` so the second one gets mapped to `train_1`.

**Comment by Jasonnor** [comment]
Hi @nfelt, thanks for the reply, but I still get same problem after change scalar name `train` to `train_acc` and `train_f1_score` or something else without same name.
![image](https://user-images.githubusercontent.com/7609173/65739989-d70e9580-e119-11e9-916f-1f87bc11d7ac.png)

Looks like a bug of summary in custom loss/metrics function with model_to_estimator.
